src/main/resources/templates/mdBenefitsFlow/homeAddress.html

```diff
@@ -10,7 +10,8 @@
       <main id="content" role="main" class="form-card spacing-above-35">
         <th:block th:replace="~{fragments/icons :: county}"></th:block>
```

Comment:
This icon should be `mail`.

Reply:
Good catch. I hadn't spotted the difference at all.